### PR TITLE
[imporove]renderdoc interceptor-lib can read Q regeisters in AArch64

### PR DESCRIPTION
**Pull request by new-TonyWang**
Interceptor-lib cannot read Q regisiters in Huawei, and there were errors in Diagnostic Log, but no more now.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -45,3 +45,6 @@ qrenderdoc/3rdparty/pyside
 .DS_Store
 plugins-*/
 __pycache__
+.vscode
+.idea
+.github
```
Comment (baldurk) [inline .gitignore:50]: Please don't add anything to the .gitignore if it's not necessary for the change.

**Review by baldurk — CHANGES_REQUESTED** [review]
This library is no longer maintained upstream so I'm fine making this change here, but please tidy up the commit history to a single commit and explain what this change does and why in the commit message so that it can be seen clearly in future.

I've also made one note about a change that should be reverted.

File: .gitignore (hunk 1 above)
Comment (new-TonyWang) [reply .gitignore:50]: ok

**Comment by baldurk** [comment]
Closing this due to lack of activity and no updates. If you would like to submit these changes again in future please open a new pull request after addressing the review feedback above.

**Pull request closed without merging**